lib/common/xml.c

```diff
@@ -5003,3 +5003,10 @@ crm_element_value(xmlNode * data, const char *name)
     }
     return (const char *)attr->children->content;
 }
+
+void
+crm_destroy_xml(gpointer data)
```

Comment:
I will keep that in mind, thanks. Just thought this function is pretty self-documenting :)